test-packages/e2e-tests/test/batch.spec.ts

```diff
@@ -107,4 +108,43 @@ describe('batch', () => {
     expect(changesetResponse.isSuccess()).toBe(true);
     expect(deleteRes.isSuccess()).toBe(true);
   });
+
+  it('deserializer should load in batches with only function imports', async () => {
+    // See https://github.com/SAP/cloud-sdk-js/issues/3539 for the original issue
+    const results = await batch(changeset(createTestEntityById({ id: 77 })))
+      .withSubRequestPathType('relativeToEntity')
+      .execute(destination);
+
+    const r = results[0];
+    if (r && r.isWriteResponses()) {
```

Comment:
[pp] I am not a fan of variables called `r` (except maybe in arrow functions - which is not the case here)
Also, some of this code can be shortened, so naming gets easier:
```suggestion
    const changesetRes = first(await batch(changeset(createTestEntityById({ id: 77 })))
      .withSubRequestPathType('relativeToEntity')
      .execute(destination))

    if (changesetRes?.isWriteResponses()) {
```

(also applies to the test below)